### PR TITLE
add winamp default shortcuts

### PR DESCRIPTION
**Pull request by soleuu**
Hello,

I have added winamp default shortcuts. 
- z : previous track
- x : play
- c : pause
- v : stop
- b : next track
- n : open

I'm having problem adding media button, I have tried those constant with no luck :
      Key_MediaTogglePlayPause
      Key_MediaPause
      Key_MediaPlay
      Key_MediaStop
      Key_Play
      Key_Stop

Is there someone who can help me ?

I'm using ubuntu 20.04.

thank you.

### CODE REVIEW

File: src/mainwindow/maindisplay.cpp [hunk 1]
```diff
@@ -337,6 +339,31 @@ MainDisplay::SetupPushButtons (const XClient* client)
...
+	connect (shortcut_pause, SIGNAL(activated()), client->xplayback (), SLOT(toggle_pause ()));
+
+	QShortcut *shortcut_stop = new QShortcut(QKeySequence("v"), this);
+	connect (shortcut_stop, SIGNAL(activated()), client->xplayback (), SLOT(stop ()));
+
+	QShortcut *shortcut_next = new QShortcut(QKeySequence("b"), this);
+	connect (shortcut_next, SIGNAL(activated()), client->xplayback (), SLOT(next ()));
+
+	QShortcut *shortcut_eject = new QShortcut(QKeySequence("n"), this);
+	connect (shortcut_eject, SIGNAL(activated()), this, SLOT(fileOpen()));
```
Comment (Malvineous) [inline src/mainwindow/maindisplay.cpp:363]: Missing spaces before `(` (e.g. `SIGNAL(` -> `SIGNAL (`, `activated()` -> `activated ()`, etc.)

File: src/playlist/playlistwidget.cpp [hunk 2]
```diff
@@ -237,6 +238,22 @@ PlaylistWidget::PlaylistWidget (PlaylistWindow *parent) : QWidget (parent)
 	connect (parent, SIGNAL (setDisplayTime (int)),
 	         m_controls, SIGNAL (setDisplayTime (int)));
 
+QShortcut *shortcut_prev = new QShortcut(QKeySequence("z"), this);
```
Comment (Malvineous) [inline src/playlist/playlistwidget.cpp:241]: Different indent

File: src/playlist/playlistwidget.cpp [hunk 3]
```diff
@@ -237,6 +238,22 @@ PlaylistWidget::PlaylistWidget (PlaylistWindow *parent) : QWidget (parent)
...
+	connect (shortcut_prev, SIGNAL(activated()), client->xplayback (), SLOT(prev ()));
+
+	QShortcut *shortcut_play = new QShortcut(QKeySequence("x"), this);
+	connect (shortcut_play, SIGNAL(activated()), client->xplayback (), SLOT(play ()));
+
+
```
Comment (Malvineous) [inline src/playlist/playlistwidget.cpp:247]: Too many blank lines

File: src/mainwindow/maindisplay.cpp [hunk 4]
```diff
@@ -337,6 +339,31 @@ MainDisplay::SetupPushButtons (const XClient* client)
...
+MainDisplay::SetupShortcuts (const XClient* client)
+{
+	/* Normal buttons */
+	QShortcut *shortcut_prev = new QShortcut(QKeySequence("z"), this);
+	connect (shortcut_prev, SIGNAL(activated()), client->xplayback (), SLOT(prev ()));
+
+	QShortcut *shortcut_play = new QShortcut(QKeySequence("x"), this);
+	connect (shortcut_play, SIGNAL(activated()), client->xplayback (), SLOT(play ()));
+
+
```
Comment (Malvineous) [inline src/mainwindow/maindisplay.cpp:352]: Extra blank line

**Review by Malvineous — CHANGES_REQUESTED** [review]
It looks fine to me apart from a few minor code style differences (not matching the existing coding style).  I don't mind merging it if these are fixed.

Can't help with the media keys though.  Personally I use my own xmms2hotkey client which makes the media keys work globally, no matter what app you're in.

File: src/mainwindow/maindisplay.cpp (hunk 1 above)
Comment (soleuu) [reply src/mainwindow/maindisplay.cpp]: patched

File: src/playlist/playlistwidget.cpp (hunk 2 above)
Comment (soleuu) [reply src/playlist/playlistwidget.cpp]: patched

File: src/mainwindow/maindisplay.cpp (hunk 4 above)
Comment (soleuu) [reply src/mainwindow/maindisplay.cpp:354]: patched

**Comment by Malvineous** [comment]
Thanks for the contribution!  Merged in 8c05cd58b922d77dcb50e29556992d3191f0a62e

**Comment by soleuu** [comment]
Thank you.
I will look for your xmms2hotkey for media button, thank you for the suggestion.
Have a nice day.

**Pull request closed without merging**